[REFACTOR] Integration of core

hyfi06 opened the pull request:
I add the work @behagoras, but gradually for not add inesesaris files and modules.
We add multiples files, configurations and modules for each step for integrate worker, but not all is necessary.

behagoras commented:
@hyfi06 It works, but I can't trace what you did, It seems to me just a squash of our code, why would we lost control of our changes?

I don't think is a good idea remove commits, It makes longer and untrazable changes, better use tags and versions, but the truth I have no professional experience manage version control.

hyfi06 commented:
I added folders of core.
- __test__
- TestManager
- templates
- worker
They have no change

I installed chai, react-app-rewired, worker-loader

I added your file config-overrides.js and Level.jsx of your last pull request
